src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1119,7 +1132,7 @@ void Transformations::MainSnippets(void) {
                 ov::is_type<ov::op::v0::Clamp>(n) || ov::is_type<ov::op::v0::Ceiling>(n) ||
                 ov::is_type<ov::op::v0::Convert>(n) || ov::is_type<ov::op::v1::Divide>(n) ||
                 ov::is_type<ov::op::v0::Elu>(n) || ov::is_type<ov::op::v0::Exp>(n) ||
-                ov::is_type<ov::op::v1::Equal>(n) || ov::is_type<ov::op::v0::FakeQuantize>(n) ||
+                ov::is_type<ov::op::v1::Equal>(n) ||
```

Comment:
Since Matmul tests are anyway disabled, does it makes any sense to disable FQ tokenizations?

Reply:
int8 roberta is a bit faster without FQ tokenization (18.9 ms vs 20 ms).
Should I revert this change?

Reply:
It is faster only with FQ Decomposition which will be done in separate PR. So yes, please revert

Reply:
Done